src/main/java/me/jellysquid/mods/lithium/mixin/world/explosions/ExplosionMixin.java

```diff
@@ -191,20 +198,25 @@ private void performRayCast(Random random, double vecX, double vecY, double vecZ
     }
 
     /**
-     * Called for every step made by a ray being cast by an explosion.
+     * Called for every step made by a ray being cast by an explosion. fai
```

Comment:
what does `fai` mean here? is it a typo? 😅

Reply:
ferks as intended

Reply:
probably me just typing by accident